ui/src/index.ts

```diff
@@ -15,16 +15,19 @@ window.addEventListener("load", async function() {
   const noCampaignPage = document.getElementById("noCampaignPage")!;
   const alreadyVotedPage = document.getElementById("alreadyVotedPage")!;
   const errorPage = document.getElementById("errorPage")!;
+  const errorUnknowUserPage = document.getElementById("errorUnknowUserPage")!;
```

Comment:
Nitpicking: I know I used "user" in the issue, however I think "employee" is a better term here since it is the term used by the code from which the error originates (see line 100). Moreover, I don't think prepending the name with "error" helps the reader. I suggest `unknownEmployeePage`.